src/applications/vaos/services/appointment/index.js

```diff
@@ -342,3 +342,23 @@ export function isAtlasLocation(appointment) {
     element => element.tasInfo,
   );
 }
+
+/**
+ * Method to check for home video appointment
+ * @param {*} appointment A FHIR appointment resource
+ * @return (Boolean} Returns whether or not the appointment is a home video appointment.
+ */
+export function isVideoHome(appointment) {
+  return appointment.contained?.[0]?.telecom?.find(
```

Comment:
I'm not positive at home video appointments are the only ones with a video url in the data we get from MAS. I know for sure that ATLAS appointments also have a link. Not sure about the other types.

I think it's safer to check the kind, like we do for clinic: https://coderepo.mobilehealth.va.gov/projects/CSS/repos/mobile-appointment-service/browse/mobile-appointment-service/src/main/resources/video-visits_v3.json#101

Our understanding is that at home appointments are MOBILE_ANY or ADHOC. So you could check that it has one of those types and isn't an ATLAS appointment.

Reply:
Done.